content/en/platform/corda/4.5/enterprise/operations/deployment/deployment-kubernetes-auth.md

```diff
@@ -35,7 +35,7 @@ helm install cenm-auth auth --set prefix=cenm --set acceptLicense=Y --set volume
 | Parameter  | Description   | Default value    |
 | ---------- | ------------- | -----------------|
 | `bashDebug`                   | Display additional information while running bash scripts (useful while investigating issues) | `false` |
-| `authImage.repository`        | URL to Network Map Docker image repository | `acrcenm.azurecr.io/auth/auth` |
+| `authImage.repository`        | URL to Network Map Docker image repository | `corda/enterprise-auth` |
```

Comment:
These `4.5/enterprise/` files should probably get the same fixes for versions, etc. from `1.3/cenm/`?